pandas/_libs/tslibs/offsets.pyx

```diff
@@ -760,13 +764,78 @@ cdef class _Tick(ABCTick):
     # ensure that reversed-ops with numpy scalars return NotImplemented
     __array_priority__ = 1000
     _adjust_dst = False
+    _inc = Timedelta(microseconds=1000)
+    _prefix = "undefined"
+    _attributes = frozenset(["n", "normalize"])
+
+    cdef readonly:
+        int64_t n
+        bint normalize
+        dict _cache
+
+    def __init__(self, n=1, normalize=False):
+        n = _BaseOffset._validate_n(n)
+        self.n = n
+        self.normalize = False
+        self._cache = {}
+        if normalize:
+            # GH#21427
+            raise ValueError(
+                "Tick offset with `normalize=True` are not allowed."
+            )
+
+    @property
+    def delta(self) -> Timedelta:
+        return self.n * self._inc
+
+    @property
+    def nanos(self) -> int64_t:
+        return self.delta.value
 
     def is_on_offset(self, dt) -> bool:
         return True
 
     def is_anchored(self) -> bool:
         return False
 
+    # --------------------------------------------------------------------
+    # Comparison and Arithmetic Methods
+
+    def __eq__(self, other):
+        if isinstance(other, str):
+            try:
+                # GH#23524 if to_offset fails, we are dealing with an
+                #  incomparable type so == is False and != is True
+                other = to_offset(other)
+            except ValueError:
+                # e.g. "infer"
+                return False
+        return self.delta == other
+
+    def __ne__(self, other):
+        if isinstance(other, str):
```

Comment:
why is this just not self.__eq__...?

Reply:
not sure, thats how it is in the status quo.  ill give it a shot and see if it breaks anything

Reply:
doesnt break anything, just pushed with this update